pkg/addon/versioned_registry.go

```diff
@@ -161,7 +161,6 @@ func (i versionedRegistry) loadAddon(ctx context.Context, name, version string)
 	if addonVersion == nil {
 		return nil, errors.Errorf("specified version %s not exist", utils.Sanitize(version))
 	}
-	klog.V(5).Infof("Addon '%s' with version '%s' found from registry '%s'", addonVersion.Name, addonVersion.Version, i.name)
```

Comment:
we'd better filter addon in helm chart format in the `chooseVersion` function instead of remove the log directly.

do we have any special label to identity an addon from helm chart structure？ \cc @wangyikewxgm

Reply:
Not yet.